platform-includes/configuration/openfeature/javascript.angular.mdx

```diff
@@ -0,0 +1,37 @@
+Before using this integration, you need to install and instrument the OpenFeature SDK in your app. Learn more by reading OpenFeature's [SDK](https://openfeature.dev/docs/reference/technologies/client/web/) and [provider](https://openfeature.dev/docs/reference/concepts/provider) docs.
+
+npm package: [`@openfeature/web-sdk`](https://www.npmjs.com/package/@openfeature/web-sdk)
+
+```JavaScript {tabTitle: JavaScript (Track All Evals)}
+import * as Sentry from '@sentry/angular';
+import { OpenFeature } from '@openfeature/web-sdk';
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [Sentry.openFeatureIntegration()]
+});
+
+OpenFeature.setProvider(new MyProviderOfChoice());
+OpenFeature.addHooks(new Sentry.OpenFeatureIntegrationHook());
```

Comment:
Is this casing right? we generally start with lowercase

```suggestion
OpenFeature.addHooks(new Sentry.openFeatureIntegrationHook());
```

Reply:
Yeah this is the current casing: https://github.com/getsentry/sentry-javascript/blob/d32bf15a0d4e499cf7d0d8e27adee35b99743f1a/packages/browser/src/integrations/featureFlags/openfeature/integration.ts#L27